_studio/mfx_lib/encode_hw/vp9/src/mfx_vp9_encode_hw_par.cpp

```diff
@@ -1479,6 +1479,15 @@ mfxStatus CheckParameters(VP9MfxVideoParam &par, ENCODE_CAPS_VP9 const &caps)
         unsupported = true;
     }
 
+#if defined(MFX_VA_LINUX)
+    // TODO: For now driver on Linux doesn't support MxN tiles
+    if (rows > 1 && cols > 1)
```

Comment:
Maybe ||? Otherwise 5x1 is supported?

Reply:
Do we have driver query caps for that? If not did you request to add them?